docs/commands/install.md

```diff
@@ -55,6 +55,20 @@ A number of applications are not currently available when using a private mirror
 - Maximo Visual Inspection
 
 
+SNO Support
+-------------------------------------------------------------------------------
```

Comment:
Review descrption as this one has been copied from Airgap, and we do not have that in SNO

Reply:
I have made the changes to the documentation to reflect that SNO will tailor the installation configuration for a connected installation.

Since the code to remove unavailable apps for SNO, the second part of the documentation can stay as is